intern/service/boss/v1/volume.py

```diff
@@ -158,8 +161,8 @@ def get_cutout(
         req = self.get_cutout_request(
             resource, 'GET', 'application/blosc',
             url_prefix, auth,
-            resolution, x_range, y_range, z_range, time_range,
-            id_list=id_list, no_cache=no_cache, **kwargs
+            resolution, x_range, y_range, z_range, time_range, access_mode.value,
```

Comment:
does `str(access_mode)` also work? this makes it more duck typing, so if the variable is not a `CacheMode` enum that it still works?

Reply:
access_mode.value in this case will be a string, and can only be one of the three options in the enum class.